audiophiler/__init__.py

```diff
@@ -217,17 +255,36 @@ def get_harold(uid, auth_dict=None):
         for auth_obj in auth_models:
             if auth_obj.auth_key == data_dict["auth_key"]:
                 harold_file_hash = None
-                harolds_list = get_harold_list(uid)
-                if len(harolds_list) == 0:
-                    harold_file_hash = get_random_harold()
+                if not get_tour_lock_status():
+                    harolds_list = get_harold_list(uid)
+                    if len(harolds_list) == 0:
+                        harold_file_hash = get_random_harold()
+                    else:
+                        harold_file_hash = random.choice(harolds_list)
                 else:
-                    harold_file_hash = random.choice(harolds_list)
-
+                    harold_file_hash = get_harold_list('root')
```

Comment:
Can't this be a list? I think you'll have to select one file hash, not a list of `Harold`s